cli/main.ml

```diff
@@ -80,6 +80,7 @@ module Device = struct
       >>= function
       | Result.Error `Disconnected -> Lwt.return_error `Disconnected
       | Result.Error `Unimplemented -> Lwt.return_error `Unimplemented
+      | Result.Error _ -> Lwt.return_error `Disconnected
```

Comment:
What are the other possibilities here? Could you just do `Result.Error x -> Lwt.return_error x` instead, or would you need to change a lot of types to do that?

Reply:
No, that doesn't work because of the types in the various Mirage interfaces, I can't even change that.